.github/workflows/CICD.yml

```diff
@@ -118,6 +118,18 @@ jobs:
         toolchain: stable
         default: true
         profile: minimal
+    - name: Print -h
+      uses: actions-rs/cargo@v1
+      with:
+        command: run
+        args: --locked -- -h
+    - name: Print --help
+      uses: actions-rs/cargo@v1
+      with:
+        command: run
+        args: --locked -- --help
+    - name: Show man page
+      run: man $(find . -name bat.1) | tee
```

Comment:
Just curious: why `man … | tee` instead of `man … | cat`? You can also do `man -l …` if you directly use it on files.

Reply:
Simply because I thought my current one-liner was sufficient since it did what I wanted and was not horrible :)

I mean, does it really matter if we pipe to `tee` or `cat`? I would argue it doesn't.

Thanks for the tip with `man -l` though, but it turns out that does not work on macOS, which I think it nice if possible.

The reason I added `| tee` in the first place was because I tested the command locally on my mac, and without it, you get into a pager, which I did not want.

Inside of CI we can simply do plain `man` though since the stdout is not a terminal, so I changed to that 👍

Reply:
> I mean, does it really matter if we pipe to `tee` or `cat`? I would argue it doesn't.

No, as I said: just curious. Thanks for the explanation.